api/src/main/java/org/pmiops/workbench/actionaudit/auditors/EgressEventAuditorImpl.kt

```diff
@@ -40,24 +40,39 @@ constructor(
 ) : EgressEventAuditor {
 
     override fun fireEgressEvent(event: SumologicEgressEvent) {
+        fireEgressEventForUser(event, null)
```

Comment:
optiona: Another option that may help improve readability. Get DbUser using 
`workspaceService.getFirecloudUserRoles` in this method. 
And pass the the right DbUser into `fireEgressEventForUser`.
Then we can keep the pretty isolated block 
```
if (user == null) {
}
```
here.